src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsMessageParser.cs

```diff
@@ -35,7 +36,7 @@ public ParseResult ParseMessage(ReadOnlyBuffer buffer, out Position consumed, ou
 
             while (buffer.Length > 0)
             {
-                if (ReadOnlyBuffer.Seek(start, end, out var lineEnd, ByteLF) == -1)
+                if (!(buffer.PositionOf(ByteLF) is SequencePosition lineEnd))
```

Comment:
Maybe flip the conditional now that it's an `is` check?

Reply:
It makes a lot of code more nested

Reply:
Ok, fair enough. Just thought that if it were quick we should do it. Go ahead and merge without changing that.